src/specter_edn/core.clj

```diff
@@ -122,13 +135,22 @@
               :remove [output-nodes
                        (rest input-nodes)
                        input-sexprs]
-              :keep   [(conj output-nodes (first input-nodes))
+              :keep   [(let [new-output (first input-nodes)]
```

Comment:
Could you extract a function here, maybe called `append-node` that handles the whitespace transparently?